config/config.go

```diff
@@ -31,7 +31,8 @@ const (
 	Follower  NodeType = "follower"
 )
 
-const DefaultLogLevels = "error;main=info;state=info;statesync=info;accumulate=debug;executor=info"
+// const DefaultLogLevels = "error;main=info;state=info;statesync=info;accumulate=debug;executor=info"
+const DefaultLogLevels = "error"
```

Comment:
Please revert this - I forgot about it. This was a change I made just to make the test log cleaner.